app/controllers/items_controller.rb

```diff
@@ -16,4 +17,25 @@ def create
       flash[:error] = "There was an error saving the item.  Please try again."
     end
   end
+
+  def destroy
+    @user = current_user
+    @item = Item.find(params[:id])
+    @item.destroy
+    flash[:notice] = "Item was removed from your list - Congratulations!"
+    redirect_to user_path(@user)
+  end
+
+  private
+
+  def item_params
+    params.require(:item).permit(:description)
+  end
+
+  def only_permit_current_user
+    unless user_signed_in? && (@user == current_user)
```

Comment:
Got it to work.

For some reason I can't get the ajax function to work properly ... removed
the `redirect_to` portion in the `destroy` method and my page isn't
refreshing without a manual command.

Talk to you in a while.

Thanks

On Mon, Jan 11, 2016 at 11:27 PM, bjblock <notifications@github.com> wrote:

> In app/controllers/items_controller.rb
> <https://github.com/aaronkoz74/blocitoff/pull/8#discussion_r49414918>:
>
> > +  def destroy
> > +    @user = current_user
> > +    @item = Item.find(params[:id])
> > +    @item.destroy
> > +    flash[:notice] = "Item was removed from your list - Congratulations!"
> > +    redirect_to user_path(@user)
> > +  end
> > +
> > +  private
> > +
> > +  def item_params
> > +    params.require(:item).permit(:description)
> > +  end
> > +
> > +  def only_permit_current_user
> > +    unless user_signed_in? && (@user == current_user)
>
> we can go over this tomorrow.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aaronkoz74/blocitoff/pull/8/files#r49414918>.
>